### ISSUE TITLE
Previous & Next buttons not accessible

### ISSUE THREAD

**Issue by CSquweb** [NUKnightLab/TimelineJS3#551]
Unable to access the Previous & Next buttons using keyboard only (and I presume a screen reader would also be affected)  As result unable to move forward and backward through the slides.

This might be helpful:  https://www.w3.org/WAI/tutorials/carousels/

**Comment by JoeGermuska** [comment]
Please clarify. I am able to visit timeline.knightlab.com, press the tab key until the contents of the embedded iframe has focus, and then use the left and right arrows to navigate the timeline.

We recognize that we have work to do to fully meet accessibility standards (#270 #343) and haven't yet scheduled the work. But in this case, if it's not clear what task is to be done.

**Comment by scott2b** [comment]
no response. closing as duplicate of above cited issues

**Comment by mgifford** [comment]
I'm not sure that this is discover-able.  How is someone supposed to know that they are on the on the slide or how to move around them. I was looking at https://timeline.knightlab.com/

It could just be that there just isn't good keyboard focus on that example site. 

I'd give this a fail for keyboard navigation given my quick test.
